src/query/exceptions.ts

```diff
@@ -0,0 +1,38 @@
+export class SelectClauseException extends Error {
+  constructor() {
+    super('The SELECT clause does not have a proper structure');
```

Comment:
Is it possible to tell why the structure is improper?

Reply:
We will update the message in the future pull request to give more detailed information to the user.